Update redis image to ubuntu/redis:latest

We are using `public.ecr.aws/lts/redis:latest` currently, but it seems there's no tags in it anymore, we should update to `public.ecr.aws/ubuntu/redis:latest` instead.
Otherwise we'll get below error when trying to use docker to develop locally.
```
xxx@yyy ~/go-sync (master) $ make docker-up
COMMIT=478f106 VERSION=v0.1.19 BUILD_TIME=1644604292 docker-compose up
Docker Compose is now in the Docker CLI, try `docker compose up`

Pulling redis (public.ecr.aws/lts/redis:latest)...
ERROR: manifest for public.ecr.aws/lts/redis:latest not found: manifest unknown: Requested image not found
make: *** [docker-up] Error 1
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -63,7 +63,7 @@ services:
     networks:
       - sync
   redis:
-    image: public.ecr.aws/lts/redis:latest
+    image: public.ecr.aws/ubuntu/redis:latest
     ports:
       - "6379:6379"
     environment:

